local-tests/send-runtime/src/main.rs

```diff
@@ -20,6 +20,10 @@ struct Args {
     #[structopt(long, short, name = "ADDRESS")]
     url: String,
 
+    /// Whether to use `wss` or `ws` protocol
+    #[structopt(long)]
+    ssl: bool,
```

Comment:
add default here (does test_update works without it?)

Reply:
compiler shouts about meaningless value - check for yourself